README.md

```diff
@@ -1,61 +1,86 @@
+
+Critically by putting the Challenge central, and managing a portfolio of Challenges via a plaform, this enables the following:
+* **Community & Capabilities**: being able to connect and grow a motivated community around the desired outcomes. Being able to find and connect to the right people who have the capabilities and are motivated to contribute. Ensuring that all citizens or organisations that wish to contribute are able to effectively do so.
+* **Collaboration**: make it easy for parties to collaborate towards the desired outcomes. Used the share understanding and community to enable low thresholds to effective collaboration.
+* **Full lifecycle**:  a Challenge is a journey with multiple stages, and needs to be managed as such. The shared representation, community and collaboration all evolve as the Challenge evolves.
+* **Best practices**: the complexity of Challenges means that we need to systematically be able to share patterns and approaches that are demonstrated to work in various situations, from phased approaches through to methods for surfacing up the inherent complexity or coming to an agreed shared understanding.
+* **Governance & Oversight**: being able to manage a portfolio of Challenges, with confidence that agreed governance / processes are being followed. Setting clear principles and boundaries for the collaboration that takes place.
+* **Trust & Transparency**: a neutral, trusted base from which all parties can confidently interact without fear of being at a disadvantage or seeing value capture unfairly by other parties.
+* **Awareness & Visibility**: being able to promote the set of Challenges within a particular context and to share in a controlled way information about both the desired outcomes and the progress towards them.
 
-Realising a platform to facilitate complex challenges is ambitious - but one that can deliver huge value...a step change in how we collaborate!
 
 If you are interested in getting involved please reach out and connect via <info@cherrytwist.org>!
 
-# How is Cherrytwist built? 
-The set of concepts and terms used for the design of CherryTwist can be found in our [Conceptual Design](./Design/ConceptualDesign.md). It is important to read this document first, as a lot of the terminology used in the rest of the design documentation as well as the code is described here - so having this mental model clear helps. 
+<p></p>
+
+
+## How do I get started?
+The easiest way to get started with Cherrytwist is to see it in action! For that there are a couple of options:
+* **[Odyssey Production instance](https://cherrytwist.odyssey.org)** - a production Cherrytwist instance showing Odyssey's hosted Challenges from their Odyssey Momentum event late 2020. 
+* **[UN SDGs Demo](https://demo-sdgs.cherrytwist.org)** - coming soon, a sample Cherrytwist instance illustrating the UN Sustainable Development Goals (SDGs)
```

Comment:
the link is broken when you click it, goes to demo-sdgs-cherrytwist.org for some reason for me...